test/XRP/signer.test.ts

```diff
@@ -70,6 +103,85 @@ describe('Signer', function (): void {
     assert.deepEqual(signedTransaction, expectedSignedTransaction)
   })
 
+  it('Sign Payment transaction with no amount', function (): void {
+    // GIVEN a Payment transaction without an amount field and a wallet.
+    const wallet = new FakeWallet(fakeSignature)
+
+    // WHEN the transaction is signed with the wallet.
+    const signedTransaction = Signer.signTransaction(
```

Comment:
took me a minute to realize that `signTransaction` does a JSON serialization first, and that the explicit serialization in the other tests is simply to generate an expected response (not necessary when we're expecting undefined or an error).  In case you notice the same thing Ted.

Reply:
thx for heads up!!